### DISCUSSION TITLE
Runtime Error "Cannot read properties of undefined (reading 'displayName')"

### DISCUSSION THREAD

**Discussion by mallidiyaan** [AntlerVC/firetable#1122] [Support / Q&A] [answered]
created repo and configured as per the document provided in github, followed the video content provided in github. i have also created a table in rowy. Help me find a solution for this error

**Comment by BeeBombshell** [comment]
Hey @mallidiyaan, exactly when are you receiving this error? Did you create any Derivative function or an Action script? Could you share some screenshots, or perhaps a video?

**Comment by mallidiyaan** [comment]
created a database in firebase and  added the api key and added generate a new key, both  generated from firebase. added both these keys while deploying and logged in with github. when it runs all the process and in the end it shows "runtime error". i completely followed step by step process provided in the video link that was given in the github repo by rowy roadmap. ["https://github.com/rowyio/roadmap"]
![Screenshot from 2023-02-16 11-43-01](https://user-images.githubusercontent.com/98813819/219284217-487f2bf8-e6b0-44ce-82d7-57adbe35d2e8.png)

**Comment by BeeBombshell** [comment, marked as answer]
Hey @mallidiyaan, after looking into it,  it seems like there is a small issue in handling the GitHub authentication. Our team is working on a fix, which will be out soon. Till then could you try logging in via a different method and check if that is working for you?
